qiskit/providers/ibmq/ibmqbackend.py

```diff
@@ -51,7 +43,7 @@ class IBMQBackend(BaseBackend):
     def __init__(
             self,
             configuration: BackendConfiguration,
-            provider: 'IBMQProvider',
+            provider,
```

Comment:
Decided to leave this variable without a type hint, because importing IBMQProvider, to use as a type, resulted in a cyclic import error. In the future, we plan to address errors like this. But, for now we are deciding to leave variables untyped, if type hinting them results in an error.